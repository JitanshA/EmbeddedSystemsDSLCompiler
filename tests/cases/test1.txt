int a;
bool flag = true;
