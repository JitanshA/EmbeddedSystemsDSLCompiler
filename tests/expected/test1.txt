MOV a, 0      ; Initialize int variable
MOV flag, 1   ; Initialize boolean variable to true
